doc/lua_api.txt

```diff
@@ -1968,11 +1981,15 @@ Elements
 
 * Show an inventory list if it has been sent to the client. Nothing will
   be shown if the inventory list is of size 0.
+* **Note**: The spacing between inventory places is not affected by the
```

Comment:
`inventory places` sounds a bit weird to me. Maybe `inventory slots` or something like `item rectangles` would be better. But note that I'm not a native speaker, so don't trust me.
(The same is some lines below in the other `list` element.)